tests/network/expose.go

```diff
@@ -816,3 +806,14 @@ func generateHelloWorldServer(vmi *v1.VirtualMachineInstance, testPort int, prot
 		&expect.BExp{R: console.RetValue("0")},
 	}, 60)).To(Succeed())
 }
+
+func createAndWaitForJobToSucceed(jobFactory func(host, port string) *batchv1.Job, namespace, ip, port, viaMessage string) error {
+	By(fmt.Sprintf("Starting a job which tries to reach the VMI via the %s", viaMessage))
+	virtClient := kubevirt.Client()
+	jobInstance := jobFactory(ip, port)
+	jobInstance, err := virtClient.BatchV1().Jobs(namespace).Create(context.Background(), jobInstance, metav1.CreateOptions{})
+	ExpectWithOffset(1, err).ToNot(HaveOccurred())
```

Comment:
This seems incorrect.
Please fix it by returning the error and not asserting it. There is a mix of both here (asserting and returning the errors).

Reply:
good catch, thanks!